src/RealtimeClient.ts

```diff
@@ -119,6 +124,8 @@ export default class RealtimeClient {
    * @param options.encode The function to encode outgoing messages. Defaults to JSON: (payload, callback) => callback(JSON.stringify(payload))
    * @param options.decode The function to decode incoming messages. Defaults to Serializer's decode.
    * @param options.reconnectAfterMs he optional function that returns the millsec reconnect interval. Defaults to stepped backoff off.
+   * @param options.worker Use Web Worker to set a side flow. Defaults to false.
+   * @param options.workerUrl The URL of the worker script. Defaults to https://realtime.supabase.com/worker.js that includes a heartbeat event call to keep the connection alive.
```

Comment:
What if we group this 2 params into only one of type `worker: string | bool` and default it to false?

1. If worker is a bool, and false, don't use worker
2. If worker is bool and true, use default worker url
3. if worker is string, use it as the url, this also means you wan't to use worker flow

Reply:
unsure... personally I kind of like it making it explicit and have options for all has that means "one thing, one behaviour change"

this feels that there are 3 possible behaviours attached to a single argument that would need extra doc reading vs intellisense for example